src/plugins/Webcam/index.js

```diff
@@ -151,24 +178,66 @@ module.exports = class Webcam extends Plugin {
     this.streamSrc = null
   }
 
+  oneTwoThreeSmile () {
+    return new Promise((resolve, reject) => {
+      let count = this.opts.countdown
+
+      let countDown = setInterval(() => {
+        if (!this.webcamActive) {
+          clearInterval(countDown)
+          this.captureInProgress = false
+          return Promise.reject('Webcam is not active')
+        }
+
+        if (count > 0) {
+          this.core.emit('informer', `${count}...`, 'warning', 800)
+          count--
+        } else {
+          clearInterval(countDown)
+          this.core.emit('informer', this.i18n('smile'), 'success', 1500)
+          setTimeout(() => resolve(), 1500)
+        }
+      }, 1000)
+    })
+  }
+
+  justSmile () {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => this.core.emit('informer', this.i18n('smile'), 'success', 1000), 1500)
+      setTimeout(() => resolve(), 2000)
+    })
```

Comment:
Waiting 2s by default if no countdown is configured seems long to me…I thought the old way where "Smile" appears on focus and snapshotting is immediate was pretty good. Don't feel strongly tho.